google/cloud/aiplatform/model_monitoring/alert.py

```diff
@@ -40,8 +47,19 @@ def __init__(
         self.enable_logging = enable_logging
         self.user_emails = user_emails
 
-    def as_proto(self):
-        """Returns EmailAlertConfig as a proto message."""
+    # TODO: remove config_for_bp parameter when model monitoring for batch prediction is GA
+    def as_proto(self, config_for_bp: Optional[bool] = False):
```

Comment:
This typing annotation should be just `config_for_bp: bool = False`.

Reply:
updated